micromasters/permissions.py

```diff
@@ -0,0 +1,30 @@
+"""
+Specific permissions for the micromasters app
+"""
+
+from rolepermissions.permissions import register_object_checker
+from rolepermissions.verifications import has_permission
+
+from micromasters.models import MicroMastersRole
+
+
+@register_object_checker()
+def can_advance_search(role, user, program):
+    """
+    Specific permission checker on a program
+    """
+    return (
+        has_permission(user, 'can_advance_search') and MicroMastersRole.objects.filter(
```

Comment:
Just to clarify, the permissions are Django user permissions? And the permissions are not attached to a program?